lntest/itest/lnd_test.go

```diff
@@ -14151,37 +14151,53 @@ func TestLightningNetworkDaemon(t *testing.T) {
 	aliceBobArgs := []string{
 		"--default-remote-max-htlcs=483",
 	}
-	if err = lndHarness.SetUp(aliceBobArgs); err != nil {
-		ht.Fatalf("unable to set up test lightning network: %v", err)
-	}
 
 	t.Logf("Running %v integration tests", len(testsCases))
 	for _, testCase := range testsCases {
-		logLine := fmt.Sprintf("STARTING ============ %v ============\n",
-			testCase.name)
+		testCase := testCase
 
-		err := lndHarness.EnsureConnected(
-			context.Background(), lndHarness.Alice, lndHarness.Bob,
-		)
-		require.NoError(t, err, "unable to connect alice to bob")
+		runTest := func() bool {
```

Comment:
Can we extract all of this into its own function? Would be a bit easier to read I think.

Reply:
By fixing the below issue, it's now just `t.Run` which is probably better to not extract. wdyt?

Reply:
Looks good and even smaller diff now. Nice!